### PR TITLE
Stats Cards Molecules

### PR DESCRIPTION
**Pull request by Shubham-Sharma-77**
Define stats cards molecules on different widths

### CODE REVIEW

File: stories/Molecules/Blocks/StatsCards/statscards.scss [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+    @include devicebreak(medium) {
+      margin-bottom: $spacing-80;
+    }
+  }
+
+  &__number {
+    color: $color-gray-200;
+    font-family: $Font-Family-SohneBold;
+    font-size: 80px;
```
Comment (surbhi90) [inline stories/Molecules/Blocks/StatsCards/statscards.scss:26]: @Shubham-Sharma-77 going forward can we use the font size in rem and letter spacings also

File: stories/Molecules/Blocks/StatsCards/statscards.scss [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    letter-spacing: 0.8px;
+    line-height: 87.2px;
+    margin-bottom: $spacing-24;
+    text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
+    -webkit-text-stroke: 2px $color-black;
```
Comment (surbhi90) [inline stories/Molecules/Blocks/StatsCards/statscards.scss:31]: @Shubham-Sharma-77 is this property supported on all the browsers

File: stories/Molecules/Blocks/StatsCards/statscards.scss [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+  &:hover {
+    background-color: $color-yellow;
+
+    & .statscard__number {
+      color: $color-black;
+    }
+  }
+}
+
+/* statscard end */
```
Comment (surbhi90) [inline stories/Molecules/Blocks/StatsCards/statscards.scss:217]: @Shubham-Sharma-77 update all the font size and letter spacing to rem

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Molecules/Blocks/StatsCards/StatsCards.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './statscards.scss';
+
+export const StatsCards = ({ title, number, percent, content, size }) => {
+    return (
+        <div className={['statscard', `statscard--${size}`].join(' ')}>
+            <div className='statscard__box'>
+                {title && <span className='statscard__title'>{title}</span>}
```
[the remark this answers is not in the document]
Comment (Shubham-Sharma-77) [reply stories/Molecules/Blocks/StatsCards/StatsCards.jsx:8]: @surbhi90 If we use a heading tag, we have to overwrite the style

File: stories/Molecules/Blocks/StatsCards/statscards.scss (hunk 2 above)
Comment (Shubham-Sharma-77) [reply stories/Molecules/Blocks/StatsCards/statscards.scss:31]: @surbhi90 -WebKit-text are not support in IE, so we can use text-shadow here

**Pull request merged**